packages/kit/src/exports/vite/index.js

```diff
@@ -107,10 +107,14 @@ const warning_preprocessor = {
 		if (!filename) return;
 
 		const basename = path.basename(filename);
-		if (basename.startsWith('+layout.') && !content.includes('<slot')) {
+		if (
+			basename.startsWith('+layout.') &&
+			(!content.includes('<slot') || (isSvelte5Plus() && !content.includes('{@render')))
+		) {
```

Comment:
Still not quite right — this means 'if content is missing `<slot` OR is Svelte 5 and content is missing `{@render`', which will always be true for Svelte 5 (unless both were present for some reason)

```suggestion
		const has_children = content.includes('<slot') || (isSvelte5Plus() && content.includes('{@render'));

		if (basename.startsWith('+layout.') && !has_children) {
```